wikimedia/wikipedia-ios
# Add article and user talk page empty view

**Phabricator:** https://phabricator.wikimedia.org/T311638 / https://phabricator.wikimedia.org/T310275 (partially)

### Notes
Adds the empty view to article and user talk pages. This also changes the `TalkPageAPIResponse` to consider an absence of topics as a successful call, because otherwise the decoding failure would propagate down the stack as an error. If my approach here is undesirable, I'm happy to change it.

### Test Steps
1. Open an empty user talk page (that is, one without any topics)
2. Confirm the empty view is shown, with the user's name in the text presented
3. Confirm the action button opens the new topic view
4. Open an empty article talk page (should be pretty easy to find on test wiki)
5. Confirm the empty view is shown
6. Confirm the action button opens the new topic view
